docs/examples.md

```diff
@@ -389,6 +389,22 @@ test_task:
   test_script: yarn run test
 ```
 
+### Yarn 2
+
+Yarn 2 (also known as Yarn Berry), has a different package cache location (`.yarn/cache`).
+To run tests, it would look like this:
+
+```yaml
+container:
+  image: node:latest
```

Comment:
this image doen't have Yarn 2 enabled. You'll need to run `yarn set version berry`